File: solid-principles/java/java-lsp-car/Car.java
public abstract class Car {
    protected int acceleration;
    protected int velocity;
    
    public Car(int acceleration, int velocity){
        this.acceleration = acceleration;
        this.velocity = velocity;
    }

    public int getAcceleration(){return this.acceleration;}
    public int getVelocity(){return this.velocity;}
    public void setAcceleration(int acceleration){this.acceleration = acceleration;}
    public void setVelocity(int velocity){this.velocity = velocity;}

    public abstract void power();
    public void run(){
        this.velocity += this.acceleration;
    }
}

File: solid-principles/csharp/csharp-lsp-car/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace YTM
{
    public abstract class Car
    {
        protected double acceleration;
        protected double velocity;
        
        public Car(double acceleration, double velocity){
            this.acceleration = acceleration;
            this.velocity = velocity;
        }

        public double GetAcceleration(){return this.acceleration;}
        public double GetVelocity(){return this.velocity;}
        public void SetAcceleration(double acceleration){this.acceleration = acceleration;}
        public void SetVelocity(double velocity){this.velocity = velocity;}

        public abstract void Power();
        public void Run(){
            this.velocity += this.acceleration;
        }
    }
}